### PR TITLE
Join feature

### PR DESCRIPTION
**Pull request by henriquechehad**

### CODE REVIEW

File: adapters/postgres/postgres.go [hunk 1]
```diff
@@ -113,6 +113,33 @@ func WhereByRequest(r *http.Request, initialPlaceholderID int) (whereSyntax stri
...
 
+// JoinByRequest implements join in queries
+func JoinByRequest(r *http.Request) (values []string, err error) {
+	joinValues := []string{}
+
+	u, _ := url.Parse(r.URL.String())
+	joinStatements := u.Query()["_join"]
+
+	for _, j := range joinStatements {
+		joinArgs := strings.Split(j, ":")
```
Comment (felipeweb) [inline adapters/postgres/postgres.go]: Here you need to check invalid indentifier use `chkInvaidIdentifier` method

Comment (henriquechehad) [inline adapters/postgres/postgres.go]: @felipeweb it's for check invalid chars in query, right? How about use it when the query is "completed" before execute, like using in this function https://github.com/nuveo/prest/blob/master/adapters/postgres/postgres.go#L123, then we call just once

Comment (felipeweb) [inline adapters/postgres/postgres.go]: yes, right LGTM

Comment (felipeweb) [inline adapters/postgres/postgres.go]: @henriquechehad so could please change to call check just after query is completed?

**Comment by felipeweb** [comment]
@henriquechehad could you add a test case with join and where together?

File: adapters/postgres/postgres.go (hunk 1 above)
Comment (henriquechehad) [inline adapters/postgres/postgres.go]: right!

**Comment by henriquechehad** [comment]
@felipeweb yes

**Comment by henriquechehad** [comment]
@felipeweb review pls

**Comment by felipeweb** [comment]
@henriquechehad  thank you

**Comment by avelino** [comment]
very very good, thanks @henriquechehad

**Pull request merged**